pkg/apis/migration/v1alpha1/migstorage_types.go

```diff
@@ -129,7 +129,7 @@ func (r *MigStorage) BuildBSL() *velero.BackupStorageLocation {
 	location := &velero.BackupStorageLocation{
 		ObjectMeta: metav1.ObjectMeta{
 			Labels:       r.GetCorrelationLabels(),
-			Namespace:    "velero",
+			Namespace:    VeleroNamespace,
```

Comment:
It's good that these will now use the const, regardless of what namespace velero is in.